src/XR/motionController/webXRAbstractMotionController.ts

```diff
@@ -291,7 +291,10 @@ export abstract class WebXRAbstractMotionController implements IDisposable {
          * handedness (left/right/none) of this controller
          */
         public handedness: MotionControllerHandedness,
-        _doNotLoadControllerMesh: boolean = false
+        /**
+         * @hidden
+         */
+        public _doNotLoadControllerMesh: boolean = false

```

Comment:
Probably. I was sticking with the format seen elsewhere though, so I'm tempted to not change it.

Reply:
yeah, it can get confusing, but as @rickfromwork said - since it is an optional parameter in the constructor it should stay this way ("falsy" being the default).